Scripts/CI/README_Metadata_StyleCheck/entry.py

```diff
@@ -0,0 +1,564 @@
+import os
+import argparse
+import subprocess as sp
+from os import listdir
+from os.path import isfile, join
+import json
+
+def main():
+    # Tell the program that we're using global variables
+    msg = 'Entry point of the docker to run json style check scripts.'
+    parser = argparse.ArgumentParser(description=msg)
+    parser.add_argument('-s', '--string', help='A JSON array of file paths.')
+    args = parser.parse_args()
+    cleanstring = ""
+    try:
+        for letter in args.string:
+            if letter not in '[" \']':
+                cleanstring+=letter
+        file_list = cleanstring.split(",")
+    except Exception as e:
+        print(f"Unable to split args {args.string}.\nException: {e}")
```

Comment:
What does this error mean to someone using the script? is something about the argument or directory structure incorrect (such as unexpected spaces or back slashes)

Reply:
I'm not honestly sure what case could trigger this exception. The arguments are automatically passed in by the GitHub Action so unless there is an error on GitHub's end, there isn't much room for user error. Ideally the Exception string `e` will provide more information. Still, I have added a bit more information to the error statement.